### ISSUE TITLE
Pushkin logo on npm

### ISSUE THREAD

**Issue by jessestorbeck** [pushkin-consortium/pushkin#251]
Currently, the logo doesn't display correctly for any pushkin packages on [npm](https://www.npmjs.com/package/pushkin-cli). I thought I fixed this the last few times I published updates, but it turns out the real problem is with their markdown parser and how it handles relative image paths (https://github.com/npm/marky-markdown/issues/437). Replacing the img tag with this should do the trick:

```html
<img src="https://raw.githubusercontent.com/pushkin-consortium/pushkin/master/pushkin_bw_w_text.png" height="400" width="450" alt="pushkin logo">
```
I made the change to worker in https://github.com/pushkin-consortium/pushkin-worker/pull/29, but CLI, API, and client should have their readme updated accordingly. We can also ditch the images folder for each of these packages. In the case of CLI, `"images/*"` should be removed from the `files` property in `package.json`.

**Comment by hunterschep** [comment]
Note: since we moved to a mono repo the link given above throws a 404, the correct link for the image is: 

`<img src="https://raw.githubusercontent.com/pushkin-consortium/pushkin/main/docs/img/pushkin_bw_w_text.png" height="400" width="450" alt="pushkin logo">`

**Comment by hunterschep** [comment]
@jessestorbeck @ayang21 I just fixed the image tags and opened Pr #266, once this is merge I can update the npm packages inside of main as I didn't want to npm publish from a branch
